frontend/src/components/Pages/MissionDefinitionPage/MissionDefinitionPage.tsx

```diff
@@ -180,7 +168,7 @@ const MissionDefinitionPageBody = ({ missionDefinition }: { missionDefinition: M
             </FormContainer>
 
             {isEditDialogOpen && (
-                <MissionDefinitionEditDialog
+                <AutoScheduleEditDialog
                     fieldName={selectedField}
```

Comment:
This dialog is not only used for editing AutoScheduling but also Name and Comment. Should keep a generic name that reflects that use